test_db_info_api.py

```diff
@@ -0,0 +1,24 @@
+import requests
```

Comment:
Are you planning on reworking this into a full download script? For testing, this is fine, but we should include a script that handles the full download, with specifying path, avoid redownloading, etc.

Reply:
good point, I need a download script anyways.